runway/data_types.py

```diff
@@ -646,3 +644,88 @@ def to_dict(self):
         ret = super(boolean, self).to_dict()
         ret['default'] = self.default
         return ret
+
+
+class point(BaseType):
```

Comment:
I'm wondering if we should define this as an `image_point` instead of a generic `point`? The main thought is:

* There may be some more generic coordinate space we want to account for that isn't [0, 1]
* We may want to use 3D points as well in the future

Reply:
Good idea!